gui/mod-manager.lua

```diff
@@ -401,6 +402,120 @@ function ModmanageScreen:init()
     }
 end
 
+ModlistMenu = defclass(ModlistMenu, widgets.Window)
+ModlistMenu.ATTRS {
+    view_id = "modlist_menu",
+    frame_title = "Active Modlist",
+    frame_style = gui.WINDOW_FRAME,
+
+    resize_min = { w = 30, h = 15 },
+    frame = { w = 40, t = 10, b = 15 },
```

Comment:
I suggest either a constant size or a size based on the length of the list (with some maximum). specifying `t` and `b` will result in a huge window with lots of empty space on larger screens.